crates/agglayer-aggregator-notifier/src/sp1/mod.rs

```diff
@@ -32,9 +33,14 @@ impl<I> super::AggregatorProver<I> for SP1<LocalProver>
 where
     I: Serialize,
 {
-    fn prove(&self, to_pack: Vec<I>) -> BoxFuture<'_, Result<Proof, anyhow::Error>> {
+    fn prove(
+        &self,
+        initial_state: LocalNetworkState,
+        certificate: I,
```

Comment:
This was introduced by @Freyskeyd in #46  to make the `agglayer-certificate-orchestrator` tests more flexible and agnostic of internal structures, focusing on testing the task flows. It can be revised if needed in the next PRs I think